packages/node/src/handlers.ts

```diff
@@ -315,6 +316,48 @@ export function errorHandler(options?: {
   };
 }
 
+interface SentryTrpcMiddlewareOptions {
+  attachRpcInput?: boolean;
```

Comment:
m: can we add a doc string?

Reply:
👍 8ab4c72e822c770e3a2a49e49cce43a849e51857